### DISCUSSION TITLE
Is there a lint for warning on `as _` casts?

### DISCUSSION THREAD

**Discussion by daprilik** [rust-lang/rust-clippy#8846] [Q&A] [answered]
It would be nice to lint against using `as _` (without linting against `as` casts in general). Maybe I'm blind, but is there a lint for this already?

**Comment by xFrednet** [comment, marked as answer]
I couldn't find on our lint list. You're welcome to file an issue for it. This might be a simple lint as it's similar to our [`clippy::as_conversions`](https://rust-lang.github.io/rust-clippy/master/index.html#as_conversions) lint :upside_down_face:

**Reply by daprilik** [reply]
Done! https://github.com/rust-lang/rust-clippy/issues/8847

**Reply by xFrednet** [reply]
Thank you! :upside_down_face:
